rootfs/usr/lib/frzr.d/bootloader.migration

```diff
@@ -0,0 +1,23 @@
+# Migrate bootloader away from syslinux
+post_install() {
+  # Actions after install
+  # Check if we have syslinux installed, if not exit
+  local mount_path=$1
+  local subvol=$2
+
+  if ![ -d "$mount_path"/boot/syslinux.cfg ] ; then
+    return 0
+  fi
+
+  echo "Running post install bootloader migration"
+
+  if [ -d /sys/firmware/efi/efivars ]; then
+    echo "UEFI boot detected, installing systemd-boot"
+    bootctl install
```

Comment:
After this is ran we'll need to configure the loader and entries at `/EFI/loader/loader.conf` and `/EFI/loader/entries/chimeraos-${version}.conf`.

Here is a working example for an entry:
```
title   ChimeraOS v40
linux   /chimeraos-40_1238104/vmlinuz-linux
initrd  /chimeraos-40_1238104/amd-ucode.img
initrd  /chimeraos-40_1238104/initramfs-linux.img
options root=LABEL=frzr_root nvme_load=YES rw rootflags=subvol=deployments/chimeraos-40_1238104 quiet splash loglevel=3 rd.systemd.show_status=auto rd.udev.log_priority=3
```